scaleapi/__init__.py

```diff
@@ -320,7 +320,7 @@ def tasks(self, **kwargs) -> Tasklist:
 
     def get_tasks(
         self,
-        project_name: str,
+        project_name: str = None,
```

Comment:
Can we enforce if project or batch is provided in the functions? 
I think we should still enforce one of them for a better querying/indexing coverage.

Reply:
Good idea @fatihkurtoglu 

@Alejandro-Gonzalez-Barriga let's add a quick check to make sure at least 1 of `project_name` or `batch_name` is provided. If not, raise an error!

Reply:
@Alejandro-Gonzalez-Barriga can you also add the same logic for `_process_tasks_endpoint_args` too?

Reply:
@Alejandro-Gonzalez-Barriga we can merge after this change ^ being applied, right?

Reply:
I've updated the logic for both methods